_topics/en/unannounced-channels.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Unannounced channels
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Private channels
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Unannounced channels** are LN channels that are not advertised to
+  the network for use in routing.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Most unannounced channels are believed to belong to users that simply
+  don't intend to route payments, such as users of mobile clients that
+  aren't always online to route payments.
+
+  Unannounced channels channels are sometimes called **private
+  channels** but this is [discouraged][unpublished channels delenda est]
```

Comment:
I don't fully understand z-man's argument. He says that if a channel is unannounced, then the connected party in that channel knows that all payments to/from that channel are ultimately to/from the other party. He then says that all channels should be announced. But in that case, doesn't the connected party still know that all payments to/from the other party are ultimately to/from the other party if there's only one announced channel to that pubkey? To avoid that, the other party needs to have at least two channels open *and* route payments (otherwise the spy could send probe payments via the victim and see that it never routes and only ever terminates payments).

Z-man's argument is against non-routing nodes rather than against unannounced channels.

Reply:
Good argument.  I'm going to yank the unannounced-channels topic from this PR, open a new PR with just it, and then invite Zmn to comment.  I don't want to change the name or URL of the topic after publication, so I think it's good to take our time to get it right---but there's no reason to hold up the other topics.